From 2899b15983e9b62362947c99f6ae49c2edb7aff0 Mon Sep 17 00:00:00 2001
From: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Date: Thu, 2 Aug 2018 17:54:28 +0300
Subject: [PATCH 020/741] sdk: dts: ls104x: drop smmu from the sdk device trees

commit d1c0a2dc0a3e51f2544a64f80e062bcbd48231e8 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

SMMU is not supported for the SDK version of the dpaa ethernet
drivers so remove the SMMU node from the device tree.

Signed-off-by: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts     | 14 ++++++++++++++
 .../boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts     | 14 ++++++++++++++
 2 files changed, 28 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts
index d9248da266ef..f5cc1020f244 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-sdk.dts
@@ -64,6 +64,20 @@
 &soc {
 #include "qoriq-dpaa-eth.dtsi"
 #include "qoriq-fman3-0-6oh.dtsi"
+
+pcie@3400000 {
+	/delete-property/ iommu-map;
+};
+
+pcie@3500000 {
+	/delete-property/ iommu-map;
+};
+
+pcie@3600000 {
+	/delete-property/ iommu-map;
+};
+
+/delete-node/ iommu@9000000;
 };
 
 &fman0 {
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts b/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts
index 06a7a8df97b7..517d644e8ab3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-sdk.dts
@@ -64,6 +64,20 @@
 &soc {
 #include "qoriq-dpaa-eth.dtsi"
 #include "qoriq-fman3-0-6oh.dtsi"
+
+pcie@3400000 {
+	/delete-property/ iommu-map;
+};
+
+pcie@3500000 {
+	/delete-property/ iommu-map;
+};
+
+pcie@3600000 {
+	/delete-property/ iommu-map;
+};
+
+/delete-node/ iommu@9000000;
 };
 
 &fsldpaa {
-- 
2.17.1

